x-pack/plugins/enterprise_search/server/routes/workplace_search/groups.ts

```diff
@@ -28,12 +28,9 @@ export function registerGroupsRoute({ router, enterpriseSearchRequestHandler }:
         }),
       },
     },
-    async (context, request, response) => {
-      return enterpriseSearchRequestHandler.createRequest({
-        path: '/ws/org/groups',
-        body: request.body,
-      })(context, request, response);
-    }
```

Comment:
`request.body` does not need to be passed & is not one of the args recognized by `createRequest`.

In general, primarily the only things we're manually passing to `createRequest` is custom overrides or helpers - anything else the handler is already receiving implicitly from `request` itself.

I've fixed `createRequest`'s typing so that it should now correctly flag/show errors when extra / unneeded args are passed to it.